include/eve/module/real/core/function/regular/simd/arm/neon/convert.hpp

```diff
@@ -163,14 +163,15 @@ namespace eve::detail
     else if constexpr( catin == category::int8x8 )
     {
            if constexpr( catou == category::int16x8 ) return vmovl_s8(v0);
+      else if constexpr( catou == category::int16x4 ) return vget_low_s16(vmovl_s8(v0));
       else                                            return convert_(EVE_RETARGET(simd_), v0, tgt);
     }
     //==============================================================================================
     // uint8 -> ?
     else if constexpr( catin == category::uint8x8 )
     {
            if constexpr( catou == category::uint16x8 )  return vmovl_u8(v0);
-      else if constexpr( catou == category::int8x8 )    return vreinterpret_s8_u8(v0);
```

Comment:
why is this one convert removed ? It was correct.

Reply:
Done before in a generic way:

https://github.com/jfalcou/eve/blob/6f8ecb1e2b17c7f4301ba37c5e485f48014bf959/include/eve/module/real/core/function/regular/simd/arm/neon/convert.hpp#L29-L40